package-res/reportviewer/report.html

```diff
@@ -116,7 +116,10 @@
 
 <div id="reportArea" class="pentaho-transparent">
   <div id="reportPageOutline">
-    <iframe id="reportContent" frameborder="0"></iframe>
+    <form id="hiddenReportContentForm" method="post" target="reportContentIframe">
```

Comment:
What things are listening to the form you just added? I am not talking about the iframe, which is still present, but the form. The only thing that other elements would be listening to is when the iFrame is loaded, but not when the POST of the new form is complete.

Though it is technically correct, using a form to perform simple POSTs is a little antiquated (kind of like putting inline styles in an HTML tag instead of having CSS). And for manipulating/loading content into an iFrame from the result of the POST, you can do the solution that is provided in this article (http://stackoverflow.com/questions/8226307/set-iframe-innerhtml-without-loading-page-in-it-with-jquery) which is to get the iFrame and write the result of the POST to the contentDocument of the iFrame.